pkgs/development/libraries/openmpi/default.nix

```diff
@@ -193,8 +193,8 @@ stdenv.mkDerivation (finalAttrs: {
         ];
         part2 = builtins.attrNames wrapperDataSubstitutions;
       };
-    in lib.optionalString stdenv.isLinux
-    ''
+    in
+    lib.optionalString stdenv.isLinux ''
```

Comment:
This last nixfmt commit could be squashed with it's former.

Reply:
I'll squash once we have sorted out all issues.

Reply:
> I'll squash once we have sorted out all issues.

I wouldn't want to squash all commits, as the diff noise is large for the `configureFlags` change... Perhaps if we'd fix `pmix` on Darwin the diff noise there will be negligible.